app/models/playlist.rb

```diff
@@ -60,7 +60,7 @@ def set_default_playlist
     end
   end
 
-  def set_another_playlist_to_default
+  def set_most_recently_updated_playlist_to_default
```

Comment:
renamed so that we remember that this method is in fact choosing the latest by updated_at date, since that's what the `default_scope` in line 19 prescribes

Reply:
tbh, I actually hate using `default_scope` because it is blindly applied to everything in the model and so easily hides filtering/ordering.  it is often at the root of unexpected behavior (esp for newcomers to a project).  imo, it is far better to be a little more verbose/use more lines, but be more explicit about ordering/filtering, rather than using `default_scope`

Reply:
I'm inclined to agree, it does come back to bite me later sometimes.

Reply:
Anyway I'm :+1: on the new method name.